### DISCUSSION TITLE
MacOS version?

### DISCUSSION THREAD

**Discussion by cpultz-lps** [jahnf/Projecteur#185] [Ideas]
I know that the point of your development was because there was a MacOS & Windows version, but no Linux version. However, these FANTASTIC remotes have become unusable on MacOS over the past 18 months due to lack of updates by Logitech. It now appears that you have a more stable software build for Linux than is available for MacOS.

Any chance you would port it over to MacOS for those of us who want to want to keep using the remotes (but don't know how to code software?)

Just asking. Have a great day.

**Comment by jahnf** [comment]
My first guess is that it would be easier to port to MacOS than to Windows- but then again I never owned a Mac. Without having a device to test on this is not possible.

Maybe there is a Qt developer out there on Mac? 🙂
